paratex/date_utils.py

```diff
@@ -0,0 +1,21 @@
+"""Utility methods to deal with date objects"""
+from datetime import date as Date
+from typing import Iterator, Tuple
+
+
+def months_between(start: Date, end: Date) -> Iterator[Tuple[str, str]]:
+    """Get the months between a start date and end date.
+
+    Returns:
+        Iterator over tuples containing the month and its year, respectively
+    """
```

Comment:
Eu gostaria que essa docstring dissesse se os limites sao exclusivos ou inclusivos

Reply:
```suggestion
    """Get the months from a start date to a end date.

    Returns:
        Iterator over tuples containing the month and its year, respectively
    """
```

Isso já deixaria claro que é inclusivo :)